test/org/sagebionetworks/bridge/services/ParticipantServiceTest.java

```diff
@@ -1139,11 +1151,12 @@ public void updateIdentifiersDoNotOverwriteExistingIdentifiers() {
         account.setEmailVerified(Boolean.TRUE);
         account.setPhone(TestConstants.PHONE);
         account.setPhoneVerified(Boolean.TRUE);
+        account.setExternalId(EXTERNAL_ID);
         when(accountDao.authenticate(STUDY, PHONE_PASSWORD_SIGN_IN)).thenReturn(account);
         when(accountDao.getAccount(any())).thenReturn(account);
         
         IdentifierUpdate update = new IdentifierUpdate(PHONE_PASSWORD_SIGN_IN, "updated@email.com",
-                new Phone("4082588569", "US"));
+                new Phone("4082588569", "US"), EXTERNAL_ID);
```

Comment:
Should this be a different external ID? Otherwise, we can't distinguish between "we didn't overwrite the external ID" and "we overwrote the external ID with the same value".

Reply:
Yes definitely, made that a different string